src/default_panels/GraphSubplotsPanel.js

```diff
@@ -114,6 +114,33 @@ const GraphSubplotsPanel = (props, {localize: _}) => (
           {label: _('Sinusoidal'), value: 'sinusoidal'},
         ]}
       />
+      <Radio
+        label={_('Resolution')}
+        attr="resolution"
+        options={[
+          {label: _('1:110,000,000'), value: 110},
+          {label: _('1:50,000,000'), value: 50},
+        ]}
+      />
+      <Numeric label={_('Scale')} attr="projection.scale" min={0} />
+    </PlotlySection>
+
+    <PlotlySection name={_('Map Rotation')}>
```

Comment:
I would key frame and rotation at the bottom, personally. not sure if anyone knows enough math to actually edit the rotation numbers tho

Reply:
it's there in cloud ya know, and I thought to put all relevant things together, everything below has to do with map contents...

Reply:
yeah let's make map contents the top stuff please, and leave the weird stuff at the bottom

Reply:
done

Reply:
OK but the region and projection should stay at the top :) the Frame and Rotation and Scale and Resolution should be at the bottom :)